pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -165,8 +165,7 @@ self: super: {
     then dontCheck (overrideCabal super.hakyll (drv: {
       testToolDepends = [];
     }))
-    # https://github.com/jaspervdj/hakyll/issues/491
-    else dontCheck super.hakyll;
+    else appendPatch super.hakyll ./patches/hakyll-version-bounds.patch;
```

Comment:
Please download that patch from https://patch-diff.githubusercontent.com/raw/jaspervdj/hakyll/pull/691.patch instead of checking it into the Nixpkgs repository.

Reply:
That makes sense. Fixed it.